dataactvalidator/filestreaming/csvS3Reader.py

```diff
@@ -34,11 +34,6 @@ def openFile(self, region, bucket, filename, csvSchema, bucketName, errorFilenam
         super(CsvS3Reader, self).openFile(
             region, bucket, filename, csvSchema, bucketName, errorFilename)
 
-    def downloadFile(self, region, bucket, filename, targetLocation):
-        """ After opening a file, download entire file to filename """
-        self.s3File = initializeFile(region, bucket, filename)
-        self.s3File.get_contents_to_filename(targetLocation)
-
```

Comment:
Haven't been able to figure out what this function was used for, doesn't seem to be called anywhere so removing it shouldn't break anything.